manifests/a/AprelTech/SilentInstallBuilder/Beta/6.2.5.0/AprelTech.SilentInstallBuilder.Beta.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Automatically updated by the winget bot at 2023/May/18
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AprelTech.SilentInstallBuilder.Beta
+PackageVersion: 6.2.5.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: wix
+  InstallerUrl: https://apreltech.com/Downloads/SibSetupBeta.msi

```

Comment:
```suggestion
  InstallerUrl: https://apreltech.com/Downloads/SibSetupBeta625.msi
```

Reply:
It appears that the version number has been appended to the file name for this version.